libclamav/autoit.c

```diff
@@ -761,6 +761,10 @@ static cl_error_t ea05(cli_ctx *ctx, const uint8_t *base, char *tmpd)
             cli_dbgmsg("autoit: file is compressed\n");
             if (cli_readint32(UNP.inputbuf) != 0x35304145) {
                 cli_dbgmsg("autoit: bad magic or unsupported version\n");
+                // Free this inputbuf and set back to NULL.
+                free(UNP.inputbuf);
```

Comment:
I wouldn't here, because the FREE macro checks if not NULL, and we already know it's not NULL (and have already been using it).  Coverity will complain about that.